Lib/test/test_free_threading/__init__.py

```diff
@@ -1,7 +1,8 @@
-import os
-
 from test import support
 
 
-def load_tests(*args):
-    return support.load_package_tests(os.path.dirname(__file__), *args)
+if support.Py_GIL_DISABLED:
+    import os
+
+    def load_tests(*args):
```

Comment:
Let's use a `@unittest.skipIf` to skip the tests rather than conditionally defining `load_tests`. As written, this leads to messages like:

```
0:11:59 load avg: 11.84 [395/478] test_free_threading ran no tests -- running (1): test_logging (3 min)
----------------------------------------------------------------------
Ran 0 tests in 0.000s

NO TESTS RAN
```

Which is more distracting when scanning the logs than a message about the test being skipped, like:

```
0:00:40 load avg: 3.09 [ 88/478] test_winsound skipped -- running (1): test_socket (40.7 sec)
test_winsound skipped -- No module named 'winsound'
```

Reply:
Should be fixed.